bigquery/deploy/bq-indexer.yaml.templ

```diff
@@ -1,31 +1,36 @@
-apiVersion: batch/v1
-kind: Job
+apiVersion: batch/v1beta1
```

Comment:
I don't want to make all users run a CronJob. Can you leave bq-indexer.yaml.templ/deploy-indexer.sh alone, and create bq-indexer-cronjob.yaml.templ with a comment on top:
```
Use this file to run the indexer in a cronjob.
```
I don't think it's worth mentioning anything anywhere else, like a README.